lib/src/features/feature_renderer.dart

```diff
@@ -4,9 +4,10 @@ import '../model/tile_model.dart';
 import '../themes/style.dart';
 import '../themes/theme.dart';
 import 'line_renderer.dart';
-import 'polygon_renderer.dart';
+import 'fill_renderer.dart';
 import 'symbol_line_renderer.dart';
 import 'symbol_point_renderer.dart';
+import 'symbol_polygon_renderer.dart';
```

Comment:
missing from the PR?

Reply:
Sorry about that one. Some other local stuff I was working on. Will remove from the PR